src/array/reducer/move-control.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { MoveArrayControlAction } from '../../actions';
+import { createFormArrayState } from '../../state';
+import { moveControlReducer } from './move-control';
+import { FORM_CONTROL_0_ID, FORM_CONTROL_ID, INITIAL_STATE_NESTED_GROUP } from './test-util';
+
+describe(`form array ${moveControlReducer.name}`, () => {
+  const testArrayValue = [ 0, 1, 2, 3, 4, 5 ];
+  const testArrayState = createFormArrayState(FORM_CONTROL_ID, testArrayValue);
+
+  it('should move controls forward', () => {
```

Comment:
This is only validating that the value was moved. I think this is fine, but you could consider testing for at least one other property (e.g. by marking the moved control as dirty or something).